.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Keep Docker dependencies up to date
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "Docker dependencies"
```

Comment:
Let's call this something like `dependencies` or just omit it entirely.

Reply:
Ok.